lib/package-readme-view.js

```diff
@@ -45,13 +45,42 @@ function sanitize (html) {
     checkbox.setAttribute('disabled', true)
   }
 
+  let path = require('path')
+  let urlProtocolRegex = new RegExp("^(?:[a-z]+:)?\/\/", "i")
+
+  for (const image of temporaryContainer.querySelectorAll('img')) {
+    let imageSrc = image.getAttribute('src')
+
+    let changeSrc = true
+
+    // If src contains a protocol then it must be absolute
+    if (urlProtocolRegex.test(imageSrc)) {
+      changeSrc = false
+    }
+
+    // If path is absolute on file system it must be a local file, e.g. emoji
+    if (path.isAbsolute(imageSrc)) {
+      changeSrc = false
+    }
+
+    if (changeSrc) {
+      if (path.isAbsolute(readmeSrcBase)) {
+        // If repoUrl is a local path
+        image.setAttribute('src', path.join(readmeSrcBase, imageSrc))
+      } else {
+        // If repoUrl is a URL
```

Comment:
Maybe expound on when this can happen in the comment

Reply:
Like this?
```javascript
// If repoUrl is a local path (i.e. package is installed)
// If repoUrl is a URL (i.e. package isn't installed)
```

Reply:
Done in latest commit